### PR TITLE
Tenable app doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/index.md [hunk 1]
```diff
@@ -243,12 +243,18 @@ Learn about the Sumo Logic apps for SaaS and Cloud applications.
   </div>
 </div>
 <div className="box smallbox38 card">
+  <div className="container">
+  <a href="/docs/integrations/saas-cloud/tenable"><img src={useBaseUrl('img/send-data/tenable-logo.png')} alt="icon" width="140"/><h4>Tenable</h4></a>
+  <p>Gain comprehensive visibility and actionable insights into your organization's security posture</p>
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/index.md:248]:
```suggestion
  <p>Gain comprehensive visibility and actionable insights into your organization's security posture.</p>
```

File: docs/integrations/saas-cloud/tenable.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+To set up [Cloud to Cloud Integration Tenable Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/tenable-source/) for the Tenable app, follow the instructions provided. These instructions will guide you through the process of creating a source using the Tenable source category, which you will need to use when installing the app. By following these steps, you can ensure that your Tenable app is properly integrated and configured to collect and analyze your Tenable data.
+
+## Installing the Tenable app​
+
+This section has instructions for installing the Symantec Web Security Service App for Sumo Logic.
+
+{@import ../../reuse/apps/app-install.md}
+
+## Viewing Tenable dashboards​
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/tenable.md:61]: The screenshots are broken for the dashboards "Tenable - Overview" and "Tenable - Audit Activity". When I accessed the image links directly, it gave me an "Access Denied" error. Also, the URLs for both dashboards appear to be the same. Could you make sure those image URLs are correct?

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@JV0812 the new article looks very good. The only issue I saw was for the dashboard images.

File: docs/integrations/saas-cloud/tenable.md (hunk 2 above)
Comment (ankitgoelcmu) [reply docs/integrations/saas-cloud/tenable.md:61]: Overview dashboard link should be https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Tenable/Tenable-Overview.png

**Comment by JV0812** [comment]
Thanks @jpipkin1, I have updated the URL and made them public in S3. It works for me now, can you check from your end.

**Review by jpipkin1 — APPROVED** [review]
@JV0812, the dashboard images now work for me. Thanks for fixing them!

**Pull request merged**